target_chains/solana/sdk/js/solana_utils/src/transaction.ts

```diff
@@ -435,57 +436,64 @@ export async function sendTransactions(
         : tx.signature ?? new Uint8Array()
     );
 
-    try {
-      const confirmTransactionPromise = connection.confirmTransaction(
-        {
-          signature: txSignature,
-          blockhash: blockhashResult.value.blockhash,
-          lastValidBlockHeight: blockhashResult.value.lastValidBlockHeight,
-        },
-        "confirmed"
-      );
+    const confirmTransactionPromise = connection.confirmTransaction(
```

Comment:
This diff is displayed very bad by github but it's not that complicated in reality